src/gui/vector/qgsvectorlayerproperties.cpp

```diff
@@ -136,18 +136,17 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   mActionLoadStyle = menuStyle->addAction( tr( "Load Style…" ) );
   connect( mActionLoadStyle, &QAction::triggered, this, &QgsVectorLayerProperties::loadStyle );
 
+  mActionSaveStyle = menuStyle->addAction( "Save Current Style…" );
+  connect( mActionSaveStyle, &QAction::triggered, this, &QgsVectorLayerProperties::saveStyleAs );
+  mActionSaveMultipleStyles = menuStyle->addAction( tr( "Save All Styles…" ) );
```

Comment:
```suggestion
  mActionSaveMultipleStyles = menuStyle->addAction( tr( "Save Styles…" ) );
```
The user can choose the styles to output, so let's not use "all" here (that conveys that you might output everything with no control)

Reply:
I agree !